src/main/java/rwg/config/ConfigRWG.java

```diff
@@ -16,6 +16,7 @@ public class ConfigRWG {
     public static boolean generateVillages = true;
     public static boolean generateUndergroundLakes = true;
     public static boolean generateUndergroundLavaLakes = true;
+    public static boolean generateLargeThaumcraftBiomes = true;
```

Comment:
Please, set default value of this config to false to prevent worldgen issues in existing worlds.

Reply:
Maby only change this to flase. If you change below too, it will become a double negative.